pkg/pow/miningTasker.go

```diff
@@ -132,13 +136,29 @@ func (mt *MiningTasker) GetWork(chan *Work) (*Work, bool) {
 		m2, err := mt.proxy.BatchGet([]string{valKey})
 		if err != nil {
 			mt.log.Info("Could not retrieve pricing data for current request id: %v", err)
-			return nil, false
+			//return nil, false
 		}
 		val := m2[valKey]
-		if len(val) == 0 {
+		if val == nil || len(val) == 0 {
```

Comment:
When a slice in golang is `nil` then `len(val)` also returns 0 so there is no need to have a double check.